CoreHelp/src/au/gov/asd/tac/constellation/help/utilities/toc/TOCItem.java

```diff
@@ -55,7 +55,7 @@ public String toString() {
      */
     @Override
     public boolean equals(final Object obj) {
-        return obj != null && obj instanceof TOCItem
+        return obj instanceof TOCItem
```

Comment:
More just curious what rule was being addressed. Seems odd to remove a null check in this instance

Reply:
![image](https://user-images.githubusercontent.com/73083035/141859334-43bb2603-c3e9-4bc1-9241-3d00d7b94717.png)
The explanation of the sonar rule is that null is not an instanceof anything so the null check is redundant.

Reply:
Gotcha